src/OrchardCore.Modules/OrchardCore.Resources/Startup.cs

```diff
@@ -23,6 +34,13 @@ public override void ConfigureServices(IServiceCollection serviceCollection)
             });
 
             serviceCollection.AddTransient<IConfigureOptions<ResourceManagementOptions>, ResourceManagementOptionsConfiguration>();
+
+            serviceCollection.Configure<ResourceOptions>(options =>
+            {
+                _configuration.GetSection("OrchardCore:OrchardCore_Resources").Bind(options);
+
+                options.ContentBasePath = _httpContextAccessor.HttpContext.Request.PathBase.Value;
```

Comment:
If I understand correctly, I don't think that `ContentBasePath` should be part of `ResourceOptions`.

But only part of `ResourceManagementOptions` that computes it as before from the current pathBase

Reply:
Done.